release.py

```diff
@@ -256,11 +260,105 @@ def command_stage_docs():
+ 
+
+        You can download the source and binary release (Scala 2.11 and Scala 2.12) from:        
+        https://kafka.apache.org/downloads#%(release_version)s                
+        
+        ---------------------------------------------------------------------------------------------------
+              
+        
+        Apache Kafka is a distributed streaming platform with four core APIs:
+        
+                 
+        ** The Producer API allows an application to publish a stream records to        
+        one or more Kafka topics.        
+                 
+        ** The Consumer API allows an application to subscribe to one or more        
+        topics and process the stream of records produced to them.      
+        
+        ** The Streams API allows an application to act as a stream processor,
+        consuming an input stream from one or more topics and producing an        
+        output stream to one or more output topics, effectively transforming the        
+        input streams to output streams.         
+        
+        ** The Connector API allows building and running reusable producers or        
+        consumers that connect Kafka topics to existing applications or data        
+        systems. For example, a connector to a relational database might        
+        capture every change to a table.
+         
+        
+        With these APIs, Kafka can be used for two broad classes of application: 
+        
+        ** Building real-time streaming data pipelines that reliably get data        
+        between systems or applications.         
+        
+        ** Building real-time streaming applications that transform or react        
+        to the streams of data.
+         
+        
+        Apache Kafka is in use at large and small companies worldwide, including        
+        Capital One, Goldman Sachs, ING, LinkedIn, Netflix, Pinterest, Rabobank,        
+        Target, The New York Times, Uber, Yelp, and Zalando, among others.
+
+        A big thank you for the following %(number_of_contributors)d contributors to this release!
+        
+        %(contributors)s
+        
+        We welcome your help and feedback. For more information on how to
+        report problems, and to get involved, visit the project website at
+        https://kafka.apache.org/ 
+
+        Thank you!
+        
+        
+        Regards,
+ 
+        <YOU>""" % release_announcement_data
+
+    print()
+    print("*****************************************************************")
+    print()
+    print(release_announcement_email)
+    print()
+    print("*****************************************************************")
+    print()
+    print("Use the above template to send the announcement for the release to the mailing list.")
+    print("IMPORTANT: Note that there are still some substitutions that need to be made in the template:")
```

Comment:
might want to also remind them that the email *must* come from their apache email address so it will be correctly delivered to `announce@apache.org` (see earlier comment). i.e. addressing this part of the wiki text that you are automating:

> You will need to use your apache email address to send out the email (otherwise, it won't be delivered to announce@apache.org)

Reply:
this is done

Reply:
yes, done